pkg/os/disk/api.go

```diff
@@ -343,7 +343,7 @@ func (imp DiskAPI) GetDiskStats(diskNumber uint32) (int64, error) {
 }
 
 func (imp DiskAPI) SetDiskState(diskNumber uint32, isOnline bool) error {
-	cmd := fmt.Sprintf("(Get-Disk -Number %d) | Set-Disk -IsOffline $%t", diskNumber, !isOnline)
+	cmd := fmt.Sprintf("Set-Disk -Number %d -IsOffline $%t; Set-Disk -Number %d -IsReadOnly $false", diskNumber, !isOnline, diskNumber)
```

Comment:
Would it be possible for `;` to ignore an error in the first `Set-Disk -Number %d -IsOffline $%t` command? Maybe we can split the execution into two commands or combine them with the Windows equivalent of &&

Reply:
@jrife does this `Set-Disk -Number %d -IsReadOnly $false` work? per comments here(https://github.com/kubernetes-csi/csi-proxy/issues/287#issuecomment-1412246794), it does not work.

Reply:
what about write like this? `Set-Disk -Number %d -IsOffline $%t -IsReadOnly $false`